src/python_pybind11/src/Helpers.cc

```diff
@@ -180,7 +192,28 @@ void defineMathHelpers(py::module &m)
         "Compute box volume")
    .def("gz_box_volume_v",
         &BoxVolumeV,
+<<<<<<< HEAD
```

Comment:
This is causing a build failure

Reply:
Dang it, that one must have also slipped through on those cherry-pick conflicts, thought I had got all ~15 of them... Fixed now.